src/main/java/core/basesyntax/Excavator.java

```diff
@@ -0,0 +1,14 @@
+package core.basesyntax;
+
+public class Excavator extends Machine {
+
+    @Override
+    public void doWork() {
+        System.out.println("Excavator started its work.");

```

Comment:
The message in the `doWork` method should indicate what type of `Machine` is working right now. In this case, it should be more informative, for example, 'Excavator is working.'